src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/basicshortmutpileup/BasicValidationResult.java

```diff
@@ -104,4 +118,112 @@ public String getFilters() {
     public long getNumAltSupportingReadsInNormal() {
         return numAltSupportingReadsInNormal;
     }
+
+    //----- The following two public static methods read and write files
+    public static void writeToFile(final List<BasicValidationResult> records, final File outputTable) {
+        try ( final BasicValidationResult.BasicValidationResultTableWriter writer = new BasicValidationResult.BasicValidationResultTableWriter(IOUtils.fileToPath(outputTable)) ) {
```

Comment:
Some inconsistencies in white space here and in the try statement below.

Reply:
done